opendcs-rest-api-jetty/src/main/java/org/opendcs/odcsapi/jetty/Start.java

```diff
@@ -73,41 +61,38 @@ public static void main(String[] args)
 		corsList.add(new String[] { "Access-Control-Allow-Headers", CrossOriginFilter.ALLOWED_HEADERS_PARAM });
 		corsList.add(new String[] { "Access-Control-Allow-Methods", CrossOriginFilter.ALLOWED_METHODS_PARAM });
 		corsList.add(new String[] { "Access-Control-Allow-Credentials", CrossOriginFilter.ALLOW_CREDENTIALS_PARAM });
-		
-		// Set up logging
-		String rootlogfile = ApiEnvExpander.expand("$DCSTOOL_USERDIR/jetty.log");
-		System.out.println("root logger goes to " + rootlogfile);
-		Logger rootLogger = Logger.getLogger("");
-		rootLogger.setLevel(Level.INFO);
-		while (rootLogger.getHandlers().length > 0)
-			rootLogger.removeHandler(rootLogger.getHandlers()[0]);
-		Handler rootHandler = new FileHandler(rootlogfile, 10000000, 5);
-		rootHandler.setFormatter(new SimpleFormatter());
-		rootLogger.addHandler(rootHandler);
-		rootLogger.info("================ Starting ===============");
 
-		String applogfile = ApiEnvExpander.expand("$DCSTOOL_USERDIR/odcsapi.log");
-		System.out.println("app logger goes to " + applogfile);
-		Logger appLogger = Logger.getLogger(ApiConstants.loggerName);
-		appLogger.setLevel(Level.INFO);
-		while (appLogger.getHandlers().length > 0)
-			appLogger.removeHandler(appLogger.getHandlers()[0]);
-		Handler appHandler = new FileHandler(applogfile, 10000000, 5);
-		appHandler.setFormatter(new LogFormatter());
-		appLogger.addHandler(appHandler);
+		apiCmdLineArgs.parseArgs(args);
+		String loggingFile = apiCmdLineArgs.getLoggingPropertiesFile();
+		InputStream is = null;
+		if (loggingFile == null) {
+			System.out.println("No log file passed.  Using defauts.");
+			is = Start.class.getClassLoader().
+					getResourceAsStream("logging.properties");
+		}
+		else
+		{
+			System.out.println("Logging File Passed: " + apiCmdLineArgs.getLoggingPropertiesFile());
+			File lf = new File(loggingFile);
+			is = new FileInputStream(lf);
+		}
+		LogManager.getLogManager().readConfiguration(is);
```

Comment:
This was flagged as a possible security issue.  I would like to give the user the ability to pass the logging.properties file to the start script if they don't want to use the default one in the resources folder.